src/components/timeline/Selection.js

```diff
@@ -6,31 +6,31 @@
 
 import * as React from 'react';
 import clamp from 'clamp';
-import { getContentRect } from '../../utils/css-geometry-tools';
```

Comment:
The changes after this file belongs to #2902. Could you revert these as well?

Reply:
Hello, @canova I am a bit confused that my absolute and relative path importing is correct or not? As I had been fixed it but if there any correction more to do, please let me know. Because I am concern about I had mixed up my code or not?